server/webhook.go

```diff
@@ -139,6 +146,193 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 		}
 		return c.String(http.StatusOK, strings.Join(createdMessages, "\n"))
 	})
+
+	// id is the webhookEndpointID in repository
+	// This endpoint is generated and injected into GitHub action & GitLab CI during the VCS setup.
+	g.POST("/sql-review/:id", func(c echo.Context) error {
+		body, err := io.ReadAll(c.Request().Body)
+		if err != nil {
+			return echo.NewHTTPError(http.StatusBadRequest, "Failed to read SQL review request").SetInternal(err)
+		}
+		log.Debug("SQL review request received for VCS project",
+			zap.String("webhook_endpoint_id", c.Param("id")),
+			zap.String("request", string(body)),
+		)
+
+		var request vcsSQLReviewRequest
+		if err := json.Unmarshal(body, &request); err != nil {
+			return echo.NewHTTPError(http.StatusBadRequest, "Malformed SQL review request").SetInternal(err)
+		}
+
+		filter := func(repo *api.Repository) (bool, error) {
```

Comment:
Should this be strings.HasPrefix(request.WebURL, repo.WebURL)?

Reply:
Well, actually the `repo.WebURL` is the full URL for the repository, like "https://github.com/bytebase/bytebase", but the `request.WebURL` we get from the CI environment is `https://github.com` (same for GitLab)

Reply:
Updated now